Rakefile

```diff
@@ -113,27 +114,31 @@ task :sync_consul_deps do
       first_three_joined = first_three.join("/")
 
       # in case the consul dependency is new and not in $GOPATH, do a "go get" first
-      e "go get #{first_three_joined}/..."
+      begin
+        e "go get #{first_three_joined}/..."
+      rescue
+        # sometimes "go get" fails with "no buildable go source files" but
```

Comment:
that can probably be done by the `-d` flag, I think? "no buildable go source files" sounds like it's trying to build+install